### DISCUSSION TITLE
Svelte support

### DISCUSSION THREAD

**Discussion by rchavezj** [ben-rogerson/twin.macro#301] [Q&A] [answered]
Will this library be supported in svelte?

**Comment by ben-rogerson** [comment, marked as answer]
From our discord:

![image](https://user-images.githubusercontent.com/21288568/105299756-91c57f80-5c0a-11eb-8d32-a4aa665374b7.png)

**Comment by ben-rogerson** [comment]
If you - or someone stumbling over this - would like to lead the charge, I'd be keen on knowing if it's even possible.

**Reply by rchavezj** [reply]
The only solution I can think of is creating a new library using Emotion.js + Tailwind ( https://dev.to/lpbayliss/how-and-why-i-use-emotion-with-tailwind-4elj ) into one ecosystem for every front end framework. 

Additional Resources:
https://svelte.dev/blog/svelte-css-in-js

**Reply by rchavezj** [reply]
When you made twin.macro, I'm curious if you re-created another version of Emotion.js but specifically for ReactJS to support jsx via CSS-in-JS

**Reply by ben-rogerson** [reply]
I'm not following you - do you mean, that if I were to recreate twin, would I drop emotion/styled-components so twin would operate on its own? Perhaps try [twind](https://github.com/tw-in-js/twind) if you'd like to drop emotion/styled-components.
